app/scripts/util/formatting/date-format.js

```diff
@@ -1,19 +1,46 @@
-import { StringFormat } from 'util/formatting/string-format';
-import { Locale } from 'util/locale';
+import { SettingsManager } from 'comp/settings/settings-manager';
```

Comment:
Modules in `utils` should not use stateful components from comp, perhaps we need to move this file since it's becoming stateful.